docs/superpowers/plans/20260820_1209_編章参照リンク.md

```
@@ -0,0 +1,112 @@
+# 条文中の編・章への参照をリンク化する
+
+Issue: #214（#204 から分離）
+
+## 背景
+
+#192 / #202 で条・項への参照をリンク化したが、編・章への参照はリンクにならない。
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)